spec/lib/reports/orders_and_fulfillment/order_cycle_supplier_totals_by_distributor_report_spec.rb

```diff
@@ -23,14 +23,22 @@ module OrdersAndFulfillment
         end
 
         it "generates the report" do
-          expect(report_table.length).to eq(2)
+          expect(report_table.length).to eq(6)
         end
 
         it "has a variant row under the distributor" do
           supplier = order.line_items.first.variant.product.supplier
           expect(report.rows.first.producer).to eq supplier.name
           expect(report.rows.first.hub).to eq distributor.name
         end
+
+        it "lists products sorted by name" do
+          order.line_items[0].variant.product.update(name: "Cucumber")
+          order.line_items[1].variant.product.update(name: "Apple")
+          order.line_items[2].variant.product.update(name: "Banane")
```

Comment:
It doesn't really matter but we used English product names and in English this one is called "Banana". :stuck_out_tongue_winking_eye: